content/docs/release-notes/archive/21-11-release-notes.md

```diff
@@ -0,0 +1,339 @@
+---
+linktitle: 2111.1 Release Notes
+title: Release Notes for Open 3D Engine 2111.1
+description: Find out what was delivered in O3DE 2111.1
+weight: 101
+toc: true
+---
+
+Good news, everyone! The Open 3D Foundation is proud to announce that we've delivered our first stable release of Open 3D Engine (O3DE), version 21.11.
+
+## Highlights
+
+### Open 3D Engine Binaries Installer
+
+There's now a binary installer for Windows for Open 3D Engine! Now you can try out our creative tools without building from source every time. This model - sometimes called "Open 3D Engine as an SDK" or "Open 3D Engine toolkit" - delivers pre-built binaries, DLLs, and headers which you can use for local development. For users who want to try out our tools without going through build time iterations, or are only interested in seeing how to write Gems and extensions to O3DE which don't require core source changes, the installer is an ideal new way to try out these workflows and find issues with us.
+
+For everyone who's joining us as a creative worker in simulations and games eager to try out Open 3D Engine: Please keep in mind our journey is just beginning! Don't expect to make a full, production-ready game in O3DE - yet. Right now we want our community to grow by playing with these tools and giving us feedback that we need to make O3DE a long-term success. That means if you're committed to trying the binary installer, be prepared for hiccups as you work, and be ready to file bugs (or maybe even make a code contribution!) as you encounter issues.
+
+Want to try it? [Download the Windows installer for O3DE 21.11 Stable!](https://o3debinaries.org/download/windows.html)
+
+{{< note >}}
+For this release of O3DE, we're using two version numbers due to some technical limitations of Windows that we're working on resolving. The source releases of O3DE are versioned as **2111.1**, and binary installers are versioned as **21.11 (Stable)**.
+{{< /note >}} 
+
+### Linux support
+
+Open 3D Engine is now considered to be available in preview for Linux! Don't expect to have support for everything you need quite yet, and we're still making sure that all the bugs and issues are ironed out, but you can now run O3DE client applications and runtimes - like the O3DE Editor itself - on Linux desktop.
+
+![The O3DE Editor running on an X desktop environment on Ubuntu 20.04 LTS (Focal Fossa)](/images/release-notes/2111-1/linux-desktop.png)
+
+Right now we only have official support for Debian 20.04 LTS (Focal Fossa) as a pre-built binary distribution, but many Linux systems which meet the [hardware and software requirements](/docs/welcome-guide/requirements/) can be modified to successfully run Open 3D Engine. To get started building O3DE and your projects on Linux, check out the [Linux getting started documentation](docs/welcome-guide/setup/setup-from-github/building-linux/)
```

Comment:
```suggestion
Right now we only have official support for Debian 20.04 LTS (Focal Fossa) as a pre-built binary distribution, but many Linux systems which meet the [hardware and software requirements](/docs/welcome-guide/requirements/) can be modified to successfully run Open 3D Engine. To get started building O3DE and your projects on Linux, check out the [Linux getting started documentation](/docs/welcome-guide/setup/setup-from-github/building-linux/)
```

Reply:
``Debian 20.04 LTS`` is not correct. Ubuntu is based on Debian, so they are not one in the same. Version 20.04 LTS refers to a specific Ubuntu release. Ubuntu uses debian-based packaging.
```suggestion
Right now we only have official support for Ubuntu 20.04.3 LTS (Focal Fossa) as a pre-built binary debian package. Other Linux systems which meet the [hardware and software requirements](/docs/welcome-guide/requirements/) is considered experimental and may need modifications to successfully run Open 3D Engine. To get started building O3DE and your projects on Linux, check out the [Linux getting started documentation](docs/welcome-guide/setup/setup-from-github/building-linux/)
```

Reply:
Updated spham's comment to incorporate additional fixes and rewording to mention both choices for getting started - installer and build docs.